packages/app-defaults/src/defaults/components.tsx

```diff
@@ -35,6 +36,16 @@ export function OptionallyWrapInRouter({ children }: { children: ReactNode }) {
   return <MemoryRouter>{children}</MemoryRouter>;
 }
 
+const DefaultRouter = (props: BrowserRouterProps) => (
+  <BrowserRouter
+    {...props}
+    future={{
+      v7_relativeSplatPath: true,
```

Comment:
Related to this: https://github.com/backstage/backstage/blob/13c8b9d895f12255606f66aea1a1556f83454712/packages/core-app-api/src/routing/FlatRoutes.tsx#L108 and https://github.com/backstage/backstage/blob/1a85f9d7b8c2ae48bc8c2386ade4f0e9e5cb157b/plugins/app/src/extensions/AppRoutes.tsx#L45